app/controllers/api/v1/orders_controller.rb

```diff
@@ -96,8 +96,23 @@ def destroy
         render json: {}
       end
 
+      def summary
+        active_orders = Order.orders_group_by_state(false)
```

Comment:
I'm not a huge fan of passing boolean arguments as it's not clear what they relate to.

Could something like `Order.orders_group_by_state(priority: false)` work better? Makes it clearer what the true and false are relating too.